ui/core/src/ui-core/inputs/labeled-input.scss

```diff
@@ -70,9 +70,9 @@
   /// Status colors ----------------------------------------------------------
 
   $status-colors: (
-    success:    $buic-accessory-success,
-    warning:    $buic-accessory-warning,
-    error:      $buic-accessory-alert
+    success:    $buic-foreground-success,

```

Comment:
These changes should also be made to labeled-textarea.scss, labeled-select.scss and labeled-themed-select.scss.

Reply:
Thanks. Done!